Common subdirectories: OctoPrint-1.10.1-orig/docs and OctoPrint-1.10.1/docs
Common subdirectories: OctoPrint-1.10.1-orig/.github and OctoPrint-1.10.1/.github
diff -c OctoPrint-1.10.1-orig/setup.py OctoPrint-1.10.1/setup.py
*** OctoPrint-1.10.1-orig/setup.py	2024-05-13 17:13:21.000000000 +0300
--- OctoPrint-1.10.1/setup.py	2025-01-11 17:30:30.019668865 +0200
***************
*** 23,29 ****
  # ----------------------------------------------------------------------------------------
  
  # Supported python versions
! PYTHON_REQUIRES = ">=3.7, <3.13"
  
  # Requirements for setup.py
  SETUP_REQUIRES = []
--- 23,29 ----
  # ----------------------------------------------------------------------------------------
  
  # Supported python versions
! PYTHON_REQUIRES = ">=3.7, <3.14"
  
  # Requirements for setup.py
  SETUP_REQUIRES = []
Common subdirectories: OctoPrint-1.10.1-orig/src and OctoPrint-1.10.1/src
Common subdirectories: OctoPrint-1.10.1-orig/tests and OctoPrint-1.10.1/tests
Common subdirectories: OctoPrint-1.10.1-orig/translations and OctoPrint-1.10.1/translations
Common subdirectories: OctoPrint-1.10.1-orig/.tx and OctoPrint-1.10.1/.tx
